open-company/open-company-web
# Share restyle

Card: https://trello.com/c/zRoa4U5y

Abstract: https://share.goabstract.com/e15814e8-0ac0-47a0-b625-6355a508b13a (first 3 sketches)

To test:
- go to stream view
- click the tile ellipsis menu
- click Share
- [x] do you see the share modal appearing in the top right corner of the stream tile? Good
- go to grid view
- click the tile ellipsis menu
- click Share
- [ ] do you see the share modal appearing in the top right corner of the grid tile? Good
- share via url (pick only team)
- [x] the copied url has the board slug in it? Good (is the fullscreen url)
- share via url (pick public read only)
- [x] the copied url DON'T has the board slug in it? Good (is the secure post url)
- share via email
- share via slack
